content/en/docs/concepts/overview/working-with-objects/_index.md

```diff
@@ -120,6 +120,22 @@ satisfy the StatefulSet specification.
 Different kinds of object can also have different `.status`; again, the API reference pages
 detail the structure of that `.status` field, and its content for each different type of object.
 
+### Server Side Field Validation
+
+Starting with Kubernetes v1.25, the API server offers server side
+[field validation](/docs/reference/using-api/api-concepts/#field-validation)
+that detects unrecognized or duplicate fields in an object. It provides all the functionality of `kubectl --validate` on the server side.
+
+The `kubectl` tool uses the `--validate` flag to set the level of field validation. It accepts the values `ignore`, `warn`, and `strict` while also
+accepting the values `true` (equivalent to `strict`) and `false` (equivalent to `ignore`). The default validation setting for kubectl is `--validate=true`,
+which means strict server-side field validation.
```

Comment:
`--validate=true` means strict server-side field validation, em... is there any means to do strict client side field validation?

Reply:
`--validate=true` means "strict validation using whatever mechanism is available". kubectl can still fall back to using client-side validation when talking to a server that doesn't support server-side field validation until 1.28.